src/processors/knex-processor.ts

```diff
@@ -198,4 +198,45 @@ export default class KnexProcessor<
       queryBuilder.offset(page.offset).limit(page.limit);
     }
   }
+
+  async getRelationships(op: Operation, record: HasId) {
+    const relationships = pick(
+      this.resourceClass.schema.relationships,
+      op.params.include
+    );
+
+    return promiseHashMap(relationships, key => {
+      if (relationships[key] instanceof Function) {
+        return relationships[key].call(this, record);
+      }
+
+      const relationshipSchema = this.resourceClass.schema.relationships[key];
+
+      return this.fetchRelationship(relationshipSchema, record);
+    });
+  }
+
+  async fetchRelationship(
+    relationship: ResourceSchemaRelationship,
+    record: HasId
+  ) {
+    if (relationship.belongsTo) {
+      const sourceTable = this.typeToTableName(
```

Comment:
Only nitpick is here let's do `this.resourceFor(relationship.type().type).getQuery() so we don't lose the right info